addons/dark-www/addon.json

```diff
@@ -2333,6 +2355,44 @@
         "darkForumCode": true
       }
     },
+    {
+      "name": "Scratch 2.0 (2013)",
+      "id": "scratchr2-blue",
+      "description": "The colors previously used on Scratch 2.0-styled pages",
+      "values": {
+        "page": "#fcfcfc",
+        "navbar": "#0f8bc0",
+        "box": "#ffffff",
+        "gray": "#f7f7f7",
+        "blue": "#e6f3fa",
+        "input": "#ffffff",
+        "button": "#18abeb",
+        "link": "#1aa0d8",
+        "footer": "#ececec",
+        "border": "#0000001a",
+        "darkBanners": "off",
+        "darkForumCode": false
+      }
+    },
+    {
+      "name": "Scratch 2.0 (2015)",
```

Comment:
Shouldn't this be "Scratch 3.0 (2015)"? I get that these are the scratch-www colors during version 2.0, but considering that they take effect on scratch-www, they would be the Scratch 3.0 colors at the time.

Reply:
These are the colors scratch-www used while Scratch 2.0 was the available online editor.... This is painful lol

Reply:
> These are the colors scratch-www used while Scratch 2.0 was the available online editor.... This is painful lol

LOL
https://web.archive.org/web/20151027002031/https://scratch.mit.edu/

Reply:
This color scheme was indeed used in early concepts for Scratch 3.0, such as [this one](https://github.com/scratchfoundation/scratch-gui/issues/1#issuecomment-250742655) from 2016. But the 3.0 beta was released in 2018 and the scratch-www color scheme was updated at the same time, so I don't think it makes sense to call the old one "3.0".